crates/bevy_ecs/src/schedule/mod.rs

```diff
@@ -125,7 +125,7 @@ mod tests {
         #[test]
         fn order_systems() {
```

Comment:
I think all the tests that are using checking the SystemOrder with multiple systems should remain multithreaded. They're more likely to fail in the multithreaded scenario.

i.e. test that have assert_eq!(world.resource::<SystemOrder>().0, vec![with multiple elements here]);